airbyte-persistence/job-persistence/src/main/java/io/airbyte/persistence/job/DefaultJobPersistence.java

```diff
@@ -304,10 +304,15 @@ public void succeedAttempt(final long jobId, final int attemptNumber) throws IOE
   }
 
   @Override
-  public void setAttemptTemporalWorkflowId(final long jobId, final int attemptNumber, final String temporalWorkflowId) throws IOException {
+  public void setAttemptTemporalWorkflowInfo(final long jobId,
+                                             final int attemptNumber,
+                                             final String temporalWorkflowId,
+                                             final String processingTaskQueue)
+      throws IOException {
     jobDatabase.query(ctx -> ctx.execute(
-        " UPDATE attempts SET temporal_workflow_id = ? WHERE job_id = ? AND attempt_number = ?",
+        " UPDATE attempts SET temporal_workflow_id = ? , processing_task_queue = ? WHERE job_id = ? AND attempt_number = ?",
```

Comment:
We'll need a database migration to add the new column. I'd probably call it `temporal_task_queue` to be consistent with the existing column called `temporal_workflow_id`. 

I'd write the migration and merge it as a separate PR before this current PR. 

Here's an example PR where I added a column to the attempts table awhile back: https://github.com/airbytehq/airbyte/pull/9579 

And here's a README with instructions around writing migrations: https://github.com/airbytehq/airbyte/tree/master/airbyte-db/db-lib#how-to-write-a-migration

Reply:
Oops, I had to refresh my DBeaver connection to see that this column is already added! I'll resolve this